src/dsp_tools/commands/excel2xml/excel2xml_lib.py

```diff
@@ -345,10 +345,10 @@ def append_permissions(root_element: etree._Element) -> etree._Element:
     return root_element
 
 
-def make_resource(
+def make_resource(  # noqa: D417 (undocumented-param)
     label: str,
     restype: str,
-    id: str,  # pylint: disable=redefined-builtin
+    id: str,  # noqa: A002 (builtin-argument-shadowing)
```

Comment:
those should be fixed, not ignored in my opinion, and they would be very easy to fix

Reply:
There are 2 reasons against:
 - If we change it, it is a breaking change.
 - The arguments of this function correspond to the attributes of the <resource> element:
   ```
   >>> str(make_resource(label="label", restype="restype", id="id", permissions="permissions"))
   <resource label="label" restype="restype" id="id" permissions="permissions"/>
   ```

I'm not against fixing it, I just don't have a good idea what the new name of the parameter could be?

Reply:
underscore it? _id